ads/google/page-level-experiments.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```

Comment:
i was about to ask the same thing. feel free to move to `src/experiments.js`
also, in the context of `src/experiments.js` , all the experiment is page-level, `page` prefix is no longer needed in your method names.